python/eos/tasks.py

```diff
@@ -100,10 +100,28 @@ def find_clusters(posterior, base_directory='./', threshold=2.0, K_g=1, analysis
     eos.info(f'Created mixture density with {len(density.components)} components')
     eos.data.MixtureDensity.create(output_path, density)
 
```

Comment:
Can you add a bit more space? I think two empty lines give a better visual separation between free standing functions.